website/static/js/fangorn.js

```diff
@@ -261,6 +324,11 @@ function _fangornSending(treebeard, file, xhr, formData) {
     xhr.send = function() {
         _send.call(xhr, file);
     };
+    var filesArr = treebeard.dropzone.getQueuedFiles();
+    if (filesArr.length  > 1) {
```

Comment:
Can the multimodal contain more than one message? Once @chrisseto is done with the add folder feature, will users be able to add a folder while file uploads are in progress?